src/actions/summit-actions.js

```diff
@@ -203,7 +203,7 @@ export const saveSummit = (entity) => async (dispatch, getState) => {
             type: 'success'
         };
 
-        postRequest(
+        return postRequest(
```

Comment:
add also the return on put https://github.com/fntechgit/summit-admin/blob/a87294fa76e82457fbb42888158bd34a2462dd89/src/actions/summit-actions.js#L187